src/main/java/org/myrobotlab/service/_TemplateService.java

```diff
@@ -17,24 +17,6 @@ public _TemplateService(String n, String id) {
     super(n, id);
   }
 
-  /**

```

Comment:
We can leave this so someone making a new service gets some guidance. Just remove one of the two asterisks in this top line to turn it into a regular comment instead of a Java doc

Reply:
I want to keep it out, because I don't want excessive data transfer between member variables and config.
Config is available at all times now within a service, the "apply" method is a specific event replacing config.
90% of the time its not needed, simply changing a config variable is enough to change the desired behavior of a service.  Only when a noobie wants "special" config handling might it be necessary, and that in general is something we want to avoid.